<ui version="4.0" >
 <class>ICatInvestigation</class>
 <widget class="QWidget" name="ICatInvestigation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Investigation Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="invstlabel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="invsttreeWidget" >
     <property name="maximumSize" >
      <size>
       <width>181</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoScrollMargin" >
      <number>9</number>
     </property>
     <property name="alternatingRowColors" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QTableWidget" name="invsttableWidget" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" >alternate-background-color: rgb(216, 225, 255);</string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode" >
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="loadcheckBox" >
       <property name="toolTip" >
        <string>Popsup Load Raw/Nexus Dialog </string>
       </property>
       <property name="text" >
        <string>Controlled loading </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="dataFilescheckBox" >
       <property name="toolTip" >
        <string>Load only  Raw and Nexus files</string>
       </property>
       <property name="text" >
        <string> DataFiles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="selectallButton" >
       <property name="toolTip" >
        <string>Select All files from the  list</string>
       </property>
       <property name="text" >
        <string>Select All Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton" >
       <property name="toolTip" >
        <string>Download  selected files</string>
       </property>
       <property name="text" >
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LoadButton" >
       <property name="toolTip" >
        <string>Load  selected files</string>
       </property>
       <property name="text" >
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="toolTip" >
        <string>Close Investigation data widget</string>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>invsttreeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
